cmd/neofs-lens/internal/storage/sanity.go

```diff
@@ -195,9 +207,13 @@ func checkShard(cmd *cobra.Command, sh storageShard) (int, error) {
 
 			switch id := string(sid); id {
 			case "":
-				err = checkObject(*header, sh.fsT)
+				if sh.fsT != nil {
```

Comment:
Seems to be somewhat inconsistent here when `== nil`. If we have no FSTree we should have no objects with "" ID, same goes for peapod type. This can happen only if metabase is incorrect. So we need to handle `== nil` case with an error as well.